docs/UserGuide.md

```diff
@@ -96,15 +96,25 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book, automatically sorted by their priority from HIGH to LOW.
+Shows a list of persons in the address book, automatically sorted by their priority from HIGH to LOW.
+- `list`: Shows a list of people who are current contacts (i.e. not archived)
+- `list archive/`: Shows a list of people who are archived
+- `list all/`: Shows a list of all people
 
-Format: `list`
+<box type="warning" seamless>
+
+**Caution:**
+- Should not be used with both `archive/` and `all/` concurrently, e.g. `list archive/ all/` ❌, `list archive/` ✅
+- `archive/` and `all/` should not have parameter values, e.g. `list archive/bob` ❌, `list archive/` ✅
+</box>
+
+Format: `list [archive/] [all/]`
```

Comment:
The format should be placed just below the command description.

Reply:
Noted